app/serializers/encryptable_minimal_serializer.rb

```diff
@@ -1,9 +1,18 @@
 # frozen_string_literal: true
 
 class EncryptableMinimalSerializer < ApplicationSerializer
-  attributes :id, :name, :description, :created_at, :sender_name
+  attributes :id, :name, :description, :created_at, :sender_name, :used_attrs
 
   def sender_name
     object.sender&.label
   end
+
+  def used_attrs
+    keys = [:password, :username, :token, :pin, :email, :custom_attr]
+    data = object.encrypted_data.instance_variable_get(:@data)
```

Comment:
move this whole logic to encryptable base model ... so if you have to access an instance variable from outside, something is probably not right :wink: